analysis/test/src/pointers.rs

```diff
@@ -536,6 +561,20 @@ unsafe fn main_0(mut argc: libc::c_int, mut argv: *mut *mut libc::c_char) -> lib
     test_load_value_store_value();
     let nums = &mut [2i32, 5i32, 3i32, 1i32, 6i32];
     insertion_sort(nums.len() as libc::c_int, nums as *mut libc::c_int);
+
+    let mut x: i32 = 0; 
+    let mut p_x = std::ptr::addr_of_mut!(x);
+    let pp_x: *mut *mut i32 = std::ptr::addr_of_mut!(p_x);
+
+    let mut y: i32 = 0; 
+    let mut p_y = std::ptr::addr_of_mut!(y);
+    let pp_y: *mut *mut i32 = std::ptr::addr_of_mut!(p_y);
+
+    ptrptr1_backward(true, pp_x, pp_y);
```

Comment:
@spernsteiner let me know if it's better to have a unique set of pointers for each call

Reply:
Shouldn't make much difference, I think.